frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -72,7 +74,13 @@ export const LookAndFeelSettings = ({
           <Text>{jt`These options require changing the server code. You can play around with and preview the options here. Check out the ${(
             <ExternalLink
               key="doc"
-              href={`${docsUrl}${utmTags}#customizing-the-appearance-of-static-embeds`}
+              href={`${docsUrl}?${new URLSearchParams({
```

Comment:
This is a change for
Interactive embedding upsell
Overview doc
✅ Look and Feel doc
Look and Feel font upsell